### ISSUE TITLE
Add new a ENV variable for HttpTimeout

### ISSUE THREAD

**Issue by EugeneRomanenko** [kuskoman/logstash-exporter#217]
I use logstash version 7.17.9 and logstash-exporter version v1.5.7.
I occasionally see on the Grafana dashboard that logstash was restarted, but actually, it was not.
logstash-exporter just didn't have time to request logstash:9600, because of default HttpTimeout 2s.

A simple demo with curl shows that logstash:9600 can answer in 0m11.947s, or even 0m13.763s
```
for i in {1..100}; do time curl -I http://localhost:9600/_node/stats/ && sleep 30; done
HTTP/1.1 200 OK
Content-Type: application/json
X-Content-Type-Options: nosniff
Content-Length: 35135


real	0m0.021s
user	0m0.005s
sys	0m0.003s
HTTP/1.1 200 OK
Content-Type: application/json
X-Content-Type-Options: nosniff
Content-Length: 35136


real	0m11.947s
user	0m0.003s
sys	0m0.006s
HTTP/1.1 200 OK
Content-Type: application/json
X-Content-Type-Options: nosniff
Content-Length: 35135

real	0m13.763s
user	0m0.002s
sys	0m0.007s
HTTP/1.1 200 OK
Content-Type: application/json
X-Content-Type-Options: nosniff
Content-Length: 35132
```

I have built logstash-exporter with HttpTimeout 15s and now I see gaps on the Grafana dashboard instead of restarts - it's much better.

Could you please add a new ENV variable for HttpTimeout?
Then I can set an appropriate value for our environment without changing the code.

Thanks.

**Comment by kuskoman** [comment]
@EugeneRomanenko implemented in v1 (see tag [v1.6.0](https://github.com/kuskoman/logstash-exporter/releases/tag/v1.6.0)). If there is any problem with it please either reopen this issue, or create another one. For documentation, please refer to [v1 branch](https://github.com/kuskoman/logstash-exporter/blob/v1/README.md)

**Comment by kuskoman** [comment]
cc @satk0, you can take implemantation of this issue in v2 if you want

**Comment by EugeneRomanenko** [comment]
> @EugeneRomanenko implemented in v1 (see tag [v1.6.0](https://github.com/kuskoman/logstash-exporter/releases/tag/v1.6.0)). If there is any problem with it please either reopen this issue, or create another one. For documentation, please refer to [v1 branch](https://github.com/kuskoman/logstash-exporter/blob/v1/README.md)

Thanks a lot!
